tests/test_updater_consistent_snapshot.py

```diff
@@ -117,6 +121,7 @@ def test_top_level_roles_update(self, test_case_data: Dict[str, Any]):
         sim = self._init_repo(consistent_snapshot)
         updater = self._init_updater(sim)
 
+        # pylint: disable=protected-access
```

Comment:
Maybe we want to disable this globally for all tests?

Reply:
I don't think that is possible as of today.
See issue https://github.com/PyCQA/pylint/issues/618 and their project where they keep track of the progress on that issue https://github.com/PyCQA/pylint/projects/1

Reply:
I don't think that is possible as of today.
See issue https://github.com/PyCQA/pylint/issues/618 and their project where they keep track of the progress on that issue https://github.com/PyCQA/pylint/projects/1

Reply:
You could call pylint separately for tests and for everything else in tox.ini, as we [do for in-toto](https://github.com/in-toto/in-toto/blob/cb562165b2f62d1fabc80b0b5fdcf212df8fd3d6/tox.ini#L29-L32).

Reply:
The only downside is that we will no longer have one source for the folders we lint or in other words we won't be able to use `lint_dirs`: https://github.com/theupdateframework/python-tuf/blob/aebd8b0fafdd52e5d6765f4dd628df0492179c1a/tox.ini#L41 for pylint.
@jku any thoughts here as you suggested `lint_dirs`?